src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -21,16 +21,18 @@
     private String mergeRemote;
     private String mergeTarget;
     private String mergeStrategy;
+    private String fastForwardMode;
 
     @DataBoundConstructor
-    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy) {
+    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy, String fastForwardMode) {
```

Comment:
For backward compatibility keep old constructor, Deprecate it, mark `@deprecated` javadoc and call new constructor with default for git merge option.